enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,269 @@
+   `ControlPlaneTopology: ArbiterHighlyAvailable` to the infrastructure config
+   object.
+7. The user then begins the install via `openshift-install create cluster`
+
+#### During Install
+
+1. The CEO will watch for new masters and the arbiter role
+2. CEO will create the operand for the etcd deployments that have tolerations
+   for the arbiter
+3. Operators that have tolerations for the arbiter should be scheduled on the
+   node
+4. The install should proceed as normal
+
+### API Extensions
+
+The `config.infrastructure.controlPlaneTopology` enum will be extended to
+include `ArbiterHighlyAvailable`
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+At the time being there is no impact on Hypershift since this edge deployment
+will require running the control plane.
+
+#### Standalone Clusters
+
+This change is relevant to standalone deployments of OpenShift at the edge or
+datacenters. This enhancement specifically deals with this type of deployment.
+
+#### Single-node Deployments or MicroShift
+
+This change does not effect Single Node or MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+Currently there are some behavior unknowns, we will need to put out a POC to
+validate some of the desires in this proposal. In it's current version this
+proposal is not exhaustive but will be filled out as we implement these goals.
+
+We currently expect this feature to mainly be used by `baremetal` installs, or
+specialized hardware that is built to take advantage of this type of
+configuration. In the current design we make two paths for cloud and baremetal
+installs in the installer. However, the cloud install is primarily for testing,
+this might mean that we simplify the installer changes if we are the only ones
+using cloud installs, since we can simply alter the manifests in the pipeline
+with out needing to change the installer.
+
+### Risks and Mitigations
+
+The main risk in this enhancement is that because we are treating one of the
+master nodes in a 3 node cluster as an arbiter, we are explicitly evicting
+processes that would otherwise be a normal supported upstream configuration such
+as a compact cluster. We run the risk of new components being critical to HA not
+containing the proper tolerations for running on the arbiter node. One of the
+mitigations we can take against that is to make sure we are testing installs and
+updates.
+
+Another risk we run is customers using an arbiter node with improper disk speeds
+below that recommended for etcd, since etcd is sensitive to latency between
+members, we should provide proper guidance so that the arbiter node doesn't
+become a bottleneck for etcd.
+
+### Drawbacks
+
+A few drawbacks we have is that we will be creating a new variant of OpenShift
+that implements a new unique way of doing HA for kubernetes. This does mean an
+increase in the test matrix and all together a different type of tests since
```

Comment:
`...and all together a different type of tests since` 

since what?

Reply:
Ops, thanks! Fixed